PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
Actually, it's code inside the .NET framework itself to handle correctly some different complex cases like shadow copies, network drives, etc, but it's not exposed for this case. We "could" use the .Location property without any kind of processing, but then not all of the cases would be handled, although I guess we are not going to use PluginCore in something like a web server?

Anyway, what is your proposal?